### ISSUE TITLE
Apache Kafka Channel Example(docs/eventing/samples/kafka/channel/index.md)

### ISSUE THREAD

**Issue by mjschmidt** [knative/docs#2481]
## Expected Behavior
`kubectl label namespace default knative-eventing-injection=enabled`
Should result in the following channels being created:
```...
knative-messaging-kafka.default.default-kn2-ingress
knative-messaging-kafka.default.default-kn2-trigger
```

## Actual Behavior
These topics are not created:
```
kubectl -n kafka exec -it my-cluster-kafka-0 -- bin/kafka-topics.sh --zookeeper localhost:2181 --list`
The resulting created kafka channels are:

Defaulting container name to kafka.
Use 'kubectl describe pod/my-cluster-kafka-0 -n kafka' to see all of the containers in this pod.
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
__consumer_offsets
knative-messaging-kafka.default.my-kafka-channel
knative-messaging-kafka.default.testchannel-one
my-topic
```

This results in the logs at the end of the example from this command being empty and the example ultimately fails:
`kubectl logs --selector='serving.knative.dev/service=broker-kafka-display' -c user-container`

I also tried to create the service account and try that as well but it did not work. What can I look at next?

**Install information:**
 - Platform (GKE, IKS, AKS, etc.): Rancher on prem deployment with rancher's istio installation
 - Knative Version: 1.14.0
-  Kubernetes version 1.15.x

**Comment by erekgit** [comment]
@mjschmidt I am also experiencing the same issue.  the ingress and trigger topics are not being created.

**Comment by erekgit** [comment]
@mjschmidt Does this work with Knative 0.13?

**Comment by mjschmidt** [comment]
I do not know it didn't work in knative 11, but I have not tried 13. I keep checking back periodically.

**Comment by medenzon** [comment]
@erekgit I’m seeing the same behavior

**Comment by mjschmidt** [comment]
I am glad that other people are seeing this issue. I wish I knew more and where to look and could help more

**Comment by erekgit** [comment]
@mjschmidt can you show the output from the logs of your default-broker-filter?

kubectl logs default-broker-filter-XXXXX

**Comment by mjschmidt** [comment]
I cannot I took down the deployment for now. I can set it back up later and get back to you though! @erekgit

**Comment by erekgit** [comment]
@mjschmidt I was able to get the Channel and Sourcing to work, will draft up my results that are appropriate for public release.   

1.)  I'd recommend installing Istio with Automatic Sidecar Injection Disabled by Default

(See if this is enough to get you in working order ^^^ but i still do not see the ingress and trigger topics being automatically created)

2.)  The Sourcing has an issue with the event_display image being broken.  Found another GitHub Issue where they're referencing a different image revision.
